### PR TITLE
fix: Move compute_small protos under its own directory

### PR DESCRIPTION
**Pull request by vam-google**
This is needed to fix API index.
Also removed gapic_yaml file, as it is not needed anymore.

### CODE REVIEW

**Review by alexander-fenster — APPROVED** [review]

**Review by jskeet — APPROVED** [review]
Thanks Vadym!

**Comment by vchudnov-g** [comment]
I suggest adding  a README to the small directory explaining what it's for: it's not the full API and the files there are just to make development easier.

**Comment by vam-google** [comment]
@vchudnov-g Added README.md, PTAL.

File: google/cloud/compute/v1small/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Google Compute Engine Minimal API (for Testing Purposes Only)
+
+Minimal viable set of API methods from [GCE](https://cloud.google.com/error-reporting/reference/).
```
Comment (vchudnov-g) [inline google/cloud/compute/v1small/README.md:3]: I would spell out Google Compute Engine

File: google/cloud/compute/v1small/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Google Compute Engine Minimal API (for Testing Purposes Only)
+
+Minimal viable set of API methods from [GCE](https://cloud.google.com/error-reporting/reference/). 
+
+These API definitions exist only for testing and to simplyfy development of
```
Comment (vchudnov-g) [inline google/cloud/compute/v1small/README.md:5]: sp: simplify

**Review by vchudnov-g — APPROVED** [review]
Thanks for doing this. Two minor comments.

File: google/cloud/compute/v1small/README.md (hunk 1 above)
Comment (vam-google) [reply google/cloud/compute/v1small/README.md]: Changed

File: google/cloud/compute/v1small/README.md (hunk 2 above)
Comment (vam-google) [reply google/cloud/compute/v1small/README.md]: fixed

**Pull request merged**